composer.json

```diff
@@ -69,7 +69,9 @@
     },
     "extra": {
         "laravel": {
-            "dont-discover": []
+            "dont-discover": [
+                "barryvdh/laravel-debugbar"
```

Comment:
ini malah akan menyebabkan tidak terload secara otomatis

Reply:
ini cuma backup dari saya,
belum final masih

Reply:
nanti jika sudah, ku buka "Ready for review" ...